### PR TITLE
[AR] Translation of index.md

### PR DESCRIPTION
**Pull request by anasselhoud**

### CODE REVIEW

File: docs/ar/index.md [hunk 1]
```diff
@@ -0,0 +1,288 @@
+---
+layout: default
+title: DEEP LEARNING
```
Comment (Atcold) [inline docs/ar/index.md:3]: Please, translate the title.

File: docs/ar/index.md [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+author: Alfredo Canziani
+lang-ref: home
+---
```
Comment (Atcold) [inline docs/ar/index.md:6]: `lang: ar` is missing. My script does not know what language this is supposed to be.

Comment (Atcold) [reply docs/ar/index.md:6]: See for example the Turkish version:

```
atcold@AlfMAC2 ~/Work/GitHub/pDL/docs [master*]
$ head tr/index.md 
---
layout: default
title: DERIN ÖĞRENME
author: Alfredo Canziani
lang-ref: home
lang: tr
translation-date: 1 July 2020
translator: Murat Ekici
---
```

File: docs/ar/index.md (hunk 1 above)
Comment (anasselhoud) [reply docs/ar/index.md:3]: Okay. Done.

File: docs/ar/index.md (hunk 2 above)
Comment (anasselhoud) [reply docs/ar/index.md:6]: Done.

**Comment by anasselhoud** [comment]
@Atcold  Please check if you can see the changes.

File: docs/ar/README-AR.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+<!-- This notebook repository now has a [companion website](https://atcold.github.io/pytorch-Deep-Learning/), where all the course material can be found in video and textual format.
+-->
+يحتوي دفتر الملاحظات هذا الآن على [موقع ويب مصاحب](https://atcold.github.io/pytorch-Deep-Learning/es/), حيث يمكن العثور على جميع مواد الدورة في شكل فيديو ونص.
+
+<!-- English - Mandarin - Korean - Spanish - Italian - Turkish - Japanese - العربية -->
+[🇬🇧](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/README.md) &nbsp; [🇨🇳](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/zh/README-ZH.md) &nbsp; [🇰🇷](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ko/README-KO.md) &nbsp; [🇪🇸](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/es/README-ES.md) &nbsp; [🇮🇹](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/it/README-IT.md) &nbsp; [🇹🇷](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/tr/README-TR.md) &nbsp; [🇯🇵](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ja/README-JA.md)
+[🇸🇦](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ar/README-AR.md)
```
Comment (Atcold) [inline docs/ar/README-AR.md:11]: Use "Arabic" in the comment.
Update the links of `README.md` for all the other languages.

Comment (Atcold) [reply docs/ar/README-AR.md:11]: Also, use `&nbsp;` to separate the flag from the Japanese one.

**Comment by anasselhoud** [comment]
@Atcold  I made the changes and added the flag to all the README files. Done.

File: README.md [hunk 4]
```diff
@@ -28,10 +28,16 @@ This notebook repository now has a [companion website](https://atcold.github.io/
...
 -->
 
+<!-- Arabic -->
+<!--
+🇸🇦 النسخة العربية من ملف `README.md`  [ar](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ar/README-AR.md) والموقع الالكتروني.
+-->
+
+
+<!-- English - Mandarin - Korean - Spanish - Italian - Turkish - Japanese - Arabic -->
+<!-- English - Mandarin - Korean - Spanish - Italian - Turkish - Japanese - Arabic -->
```
Comment (Atcold) [inline README.md:38]: Why the same line twice?

File: README.md [hunk 5]
```diff
@@ -28,10 +28,16 @@ This notebook repository now has a [companion website](https://atcold.github.io/
 🇯🇵 日本語版の `README.md` は  [ここ](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ja/README-JA.md) にあります.
 -->
 
+<!-- Arabic -->
+<!--
+🇸🇦 النسخة العربية من ملف `README.md`  [ar](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ar/README-AR.md) والموقع الالكتروني.
+-->
+
+
+<!-- English - Mandarin - Korean - Spanish - Italian - Turkish - Japanese - Arabic -->
```
Comment (Atcold) [inline README.md:37]:
```suggestion

```

**Review by Atcold — APPROVED** [review]
Thanks!

**Pull request merged**